src/common/utils.js

```diff
@@ -70,3 +70,39 @@ export async function takePicture (format) {
     })
   })
 }
+
+export function parseCSV (str) {
```

Comment:
Please move it to your plugin code. CSV is quite flexible format and doesn't specify standard separator. Widely used parsing function must take it into consideration or be just a helper for its plugin.

Reply:
done